modules/core/test/resources/META-INF/smithy/traits.smithy

```diff
@@ -176,3 +178,13 @@ enum TestOpenEnum {
 intEnum TestOpenIntEnum {
     ONE = 1
 }
+
+structure TestUrlFormFlattened {
+    @urlFormFlattened
+    test: StringList
+}
+
+structure TestUrlFormName {
+    @urlFormName("Test")
```

Comment:
this should be accompanied with a smoke test ensuring that you can retrieve the `"Test"` value